build.sbt

```diff
@@ -137,7 +137,6 @@ lazy val chisel = (project in file(".")).
   // Prevent separate JARs from being generated for coreMacros and chiselFrontend.
   dependsOn(coreMacros % "compile-internal;test-internal").
   dependsOn(chiselFrontend % "compile-internal;test-internal").
-  aggregate(coreMacros, chiselFrontend).
```

Comment:
Is the comment `// Prevent separate JARs from being generated for coreMacros and chiselFrontend.` still true with this line deleted? Why does it matter if they're in separate JARs?

Reply:
It's cleaner. It minimizes the possibility that someone will end up with jars from different chisel3 versions and have things break in ways we never imagined.